### DISCUSSION TITLE
Haven't been able to deploy  for months and pay for vercel pro

### DISCUSSION THREAD

**Discussion by thereisryan** [vercel/community#4174] [Account Issue] [answered]
### Question

Hey there Vercel team,

I no longer use an email connected to an account, that is linked to a mobile phone that I am unable to unlink without actually deleting everything that I own. This has been extremely frustrating, and now for some reason your project caching thinks that I don't have a pro account when trying to log a support case....

Maybe this way I can actually get some traction? I don't actively check the email and I would like to have it transfered over, and I guess use my github SSO. When trying to invite to the team, it sent out an invite for the free hobby plan...

**Comment by amyegan** [comment, marked as answer]
Hey @thereisryan. I think you're asking about an account verification problem, but please correct me if I misunderstood. 

If you are unable to log in, please email [registration@vercel.com](mailto:registration@vercel.com). Be sure to include some information about your account, such as the email, team name, and details of the projects you deploy with Vercel.

**Comment by thereisryan** [comment]
Will do, thanks Amy!Sent from my iPhoneOn Sep 26, 2023, at 12:30 PM, Amy ***@***.***> wrote:﻿
Hey @thereisryan. I think you're asking about an account verification problem, but please correct me if I misunderstood.
If you are unable to log in, please email ***@***.*** Be sure to include some information about your account, such as the email, team name, and details of the projects you deploy with Vercel.
